package/yast2.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Thu Apr 12 12:01:49 UTC 2018 - knut.anderssen@suse.com
+
+- Wizard: Fall back to smaller and/or hicolor icon if the icon for
+  the launched module can not be found (bsc#1087224)
```

Comment:
just small note, I think that it should be together `cannot`

Reply:
Correct, it's `cannot` here.
Officially you only use `can not` if the `not` is part of the word after it; for example as `not only` in `can not only`.
However. `can not` is acceptable.